### PR TITLE
Updating parameter name validation regex

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

**Comment by soninaren** [comment]
@fabiocav can you take a look?

File: Bindings/bindings.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
                     "help": "$timerTrigger_name_help",
                     "validators": [
                         {
-                            "expression": "^[a-zA-Z_$][a-zA-Z_$0-9]*$",
+                            "expression": "^[a-zA-Z][a-zA-Z0-9]{0,127}$",
```
Comment (mathewc) [inline Bindings/bindings.json]: Allow for 1

**Comment by mathewc** [comment]
:shi"

File: Bindings/bindings.json (hunk 1 above)
Comment (mathewc) [inline Bindings/bindings.json]: Confirmed with Fabio that this is correct

**Pull request merged**